tests/test_model_config_generator.py

```diff
@@ -859,14 +861,15 @@ def _run_and_test_model_config_generator(self,
         finally:
             self.mock_model_config.stop()
 
-    def _get_next_fake_results(self):
+    def _get_next_fake_results(self, constraint_manager):
```

Comment:
Do any of these tests actually require a constraint_manager?

Reply:
This additional change was verified and confirmed to not be required. Updated the code.